identity-service/src/routes/notifications.js

```diff
@@ -377,7 +399,7 @@ module.exports = function (app) {
    *
   */
   app.post('/notifications/all', authMiddleware, handleResponse(async (req, res, next) => {
-    let { isRead, isViewed } = req.body
+    let { isRead, isViewed, clearBadges } = req.body
```

Comment:
@raymondjacobson - This is the new flag that should clear badge counts, let me know if you need more info. 

So basically an open of the mobile app should pass both isViewed and clearBadges to the identity service

Reply:
this is perfect